docs/versions.txt

```diff
@@ -13,9 +13,9 @@ JanusGraph.
 [options="header"]
 |==========================
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop | Spark | Scala
-| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.z*,2.z,5.z |  5.3-5.5.z, 6.2-6.6.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
```

Comment:
We shouldn't change this retroactively to claim to not support the versions we supported earlier. Namely, JanusGraph previously shipped claiming to support Bigtable 0.9.5.z (and specified 0.9.z as the generic format); with the change to support literally "0.9.7.1", it implies that 0.9.5.z is/was no longer supported.

My recommendation is to change it back to 0.9.z, while adding 1.0.0-preZ as an additionally-supported version mask.

Thoughts?